include/NovelRT/Audio/AudioService.h

```diff
@@ -1,17 +1,16 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 
-#ifndef NOVELRT_NOVELAUDIOSERVICE_H
-#define NOVELRT_NOVELAUDIOSERVICE_H
+#ifndef NOVELRT_AUDIOSERVICE_H
+#define NOVELRT_AUDIOSERVICE_H
 
 #include <SDL2/SDL.h>
 #include <SDL2/SDL_mixer.h>
 #include <string>
 #include <map>
 #include <typeinfo>
-#include "NovelLoggingService.h"
 
-namespace NovelRT {
-  class NovelAudioService {
+namespace NovelRT::Audio {
+  class AudioService {
```

Comment:
nit: We aren't consistent on whether there should be an extra newline between the namespace/class or not

Reply:
Is this something we could enforce with `clang-format`?